Library/Homebrew/download_strategy.rb

```diff
@@ -184,7 +184,11 @@ def stage
       with_system_path { buffered_write("bunzip2") }
     when :gzip, :bzip2, :compress, :tar
       # Assume these are also tarred
-      with_system_path { safe_system "tar", "xf", cached_location }
+      if ARGV.verbose?
+        with_system_path { safe_system "tar", "xvf", cached_location }
+      else
+        with_system_path { safe_system "tar", "xf", cached_location }
+      end
```

Comment:
Are you talking about something like

```ruby
tar_flags = ARGV.verbose? ? "xvf" : "xf"
with_system_path { safe_system "tar", tar_flags, cached_location }
```

or are you talking about an actual array of flags? I mean, I can't think of any other flag that might be added in the future, so hardcoding `xf` or `xvf` seems fine to me.